### PR TITLE
community_meetup layer5 about

### LINKED ISSUE #2690: [About] Swap "Community First" image on About page
**Current Behavior**
The About page (https://layer5.io/company/about) has an image next to "Community First" section with this image:

**Desired Behavior**
Swap the existing image next to "Community First" section with this image: https://pbs.twimg.com/media/FO4vSXcWQAUc1Uj?format=jpg

Reference tweet: https://twitter.com/oluwabamikemi/status/1508194928442159112

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Michellebuchiokonicha**
Signed-off-by: Michellebuchiokonicha <michelleokonicha@gmail.com>

**Description**
Changed the image for the about page.

This PR fixes #2690

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit ebb4389dbe86bf0ba3e47219bcf8ddd13e6e7a49 at: https://625aee947298302bf3757b1a--layer5ng.netlify.app

**Review by Abhijay007 — COMMENTED** [review]
@Michellebuchiokonicha I think this (in file `.gitattributes) change is not needed, please revert

**Comment by Michellebuchiokonicha** [comment]
> @Michellebuchiokonicha I think this (in file `.gitattributes) change is not needed, please revert

it is not a change. I used it for the line-break error. @Nikhil-Ladha

**Comment by Michellebuchiokonicha** [comment]
> 

alright.

**Comment by Michellebuchiokonicha** [comment]
> 🚀 Preview for commit [ebb4389](https://github.com/layer5io/layer5/commit/ebb4389dbe86bf0ba3e47219bcf8ddd13e6e7a49) at: https://625aee947298302bf3757b1a--layer5ng.netlify.app

It is okay

**Comment by warunicorn19** [comment]
@Michellebuchiokonicha could you please revert the `.gitattributes` file from the commit.

**Comment by Nikhil-Ladha** [comment]
> @Michellebuchiokonicha could you please revert the `.gitattributes` file from the commit.

The change is needed because in Windows line breaks are treated differently than that of Linux.

**Comment by warunicorn19** [comment]
Ohh okay @Nikhil-Ladha gotcha

File: src/sections/Company/About/index.js [hunk 1]
```diff
@@ -59,7 +60,7 @@ const About = () => {
               </Col>
               <Col xs={12} sm={6}>
                 <div className="head-images">
-                  <img src={Tshirts} alt="free-tshirts" />
+                  <img src={Community_meetup} alt="free-tshirts" />
```
Comment (Nikhil-Ladha) [inline src/sections/Company/About/index.js:63]: Update the alt text.

File: src/sections/Company/About/index.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ import layer5BusCard from "./images/layer5-business-card-photo_med.jpg";
 import location from "./images/location.svg";
 import WhoWeAre from "../WhoWeAre";
 import FiveIcon from "../../../assets/images/layer5/5 icon/svg/light/5-light-bg.svg";
-import Tshirts from "./images/free-tshirts.jpg";
+// import Tshirts from "./images/free-tshirts.jpg";
```
Comment (Nikhil-Ladha) [inline src/sections/Company/About/index.js:12]: Remove the line instead of commenting

**Pull request closed without merging**